Typo and grammar fix (#2429)
mondoohq/cnquery: docs/development.md

## Debug providers

> In v9 we introduced providers, which split up the providers into individual go modules. This makes it more development

Reviewer: ```suggestion
In v9 we introduced provider plugins, which split up the providers into individual go modules. This makes development
```
